[BUG] AS GITHUB->Actions does not work correctly

<!-- SPDX-FileCopyrightText: © Fossology contributors

     SPDX-License-Identifier: GPL-2.0-only
-->

<!-- Before filling this issue, please read the Wiki (https://github.com/fossology/fossology/wiki)
and search if the bug do not already exists in the issues (https://github.com/fossology/fossology/issues). -->

### Description

I use the product as github->action. But for the last few months it hasn't been working. Previously it worked great. Please tell me maybe I'm doing something wrong.

#### How to reproduce

I am using this file as github->actions:

```
name: Check diff for license with Fossology

on: [pull_request]

jobs:
  check-license:
    name: Check license
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v3
      - run: |
          docker run --rm --name "fossologyscanner" -w "/opt/repo" -v ${PWD}:/opt/repo \
            -e GITHUB_TOKEN=${{ github.token }} \
            -e GITHUB_PULL_REQUEST=${{ github.event.number }} \
            -e GITHUB_REPOSITORY=${{ github.repository }} \
            -e GITHUB_REPO_URL=${{ github.repositoryUrl }} \
            -e GITHUB_REPO_OWNER=${{ github.repository_owner }} \
            -e GITHUB_API=${{ github.api_url }} \
            -e GITHUB_ACTIONS=true \
            fossology/fossology:scanner "/bin/fossologyscanner" nomos ojo

  check-copyright:
    name: Check copyright
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v3
      - run: |
          docker run --rm --name "fossologyscanner" -w "/opt/repo" -v ${PWD}:/opt/repo \
            -e GITHUB_TOKEN=${{ github.token }} \
            -e GITHUB_PULL_REQUEST=${{ github.event.number }} \
            -e GITHUB_REPOSITORY=${{ github.repository }} \
            -e GITHUB_API=${{ github.api_url }} \
            -e GITHUB_REPO_URL=${{ github.repositoryUrl }} \
            -e GITHUB_REPO_OWNER=${{ github.repository_owner }} \
            -e GITHUB_ACTIONS \
            fossology/fossology:scanner "/bin/fossologyscanner" copyright keyword
```
But every time I get an error:
```
Status: Downloaded newer image for fossology/fossology:scanner
Traceback (most recent call last):
  File "/bin/fossologyscanner", line 273, in <module>
    sys.exit(main(args))
  File "/bin/fossologyscanner", line 233, in main
    cli_options.update_args(parsed_args)
  File "/bin/FoScanner/CliOptions.py", line 71, in update_args
    self.report_format = ReportFormat[args.report]
  File "/usr/lib/python3.9/enum.py", line 408, in __getitem__
    return cls._member_map_[name]
KeyError: <ReportFormat.TEXT: 0>
Error: Process completed with exit code 1.
```

--- a/utils/automation/fossologyscanner.py
+++ b/utils/automation/fossologyscanner.py
@@ -267,7 +267,7 @@ if __name__ == "__main__":
   )
   parser.add_argument(
     "--report", type=str, help="Type of report to generate. Default 'TEXT'.",
-    choices=[member.name for member in ReportFormat], default=ReportFormat.TEXT
+    choices=[member.name for member in ReportFormat], default=ReportFormat.TEXT.name
   )
   args = parser.parse_args()
   sys.exit(main(args))

